cloudify_kubernetes/decorators.py

```diff
@@ -218,6 +220,9 @@ def wrapper(**kwargs):
             client_config_from_inputs
         )
 
+        configuration_property = create_tempfiles_for_certs_and_keys(
```

Comment:
There is a chance that after changing the config to this temp path and then remove it, the user will invoke an operation and we will try to use non existing file?